src/frontend/src/components/DateField/index.tsx

```diff
@@ -14,19 +14,41 @@ interface Props {
   formik: FormikContextType<any>;
   helperText?: any;
   onChange?: (d: ChangeEvent) => void;
+  shouldShowEditedCellsAsMarked?: boolean;
 }
 
+export type valueType = string | boolean | NamedGisaidLocation | undefined;
+
 export default function DateField({
   fieldKey,
   formik,
   helperText,
   onChange,
+  shouldShowEditedCellsAsMarked,
 }: Props): JSX.Element {
-  const { handleChange, handleBlur, values, touched, errors } = formik;
+  const { handleChange, handleBlur, values, touched, errors, initialValues } =
+    formik;
+  const [isBackgroundColorShown, setBackgroundColorShown] =
```

Comment:
Nit: I want us to have a convention for React state values and their setters that is to blindly follow the `fooBar` and `setFooBar` structure. I know it's cumbersomely long, but I think the consistency of `setIsBackgroundColorShown` is worthwhile

Reply:
+1